src/index.ts

```diff
@@ -25,6 +25,11 @@ interface PluginConfig {
      */
     buildDirectory?: string
 
+    /**
+     * The path to the "hot" file.
+     */
+    hotFile?: string
```

Comment:
Should we add a `@default` annotation to this one like the others?

It's a little tricky because it's partially computed, but I think `'public/hot'` would still make sense.

Reply:
I did consider this at the time, but decided against it because it is calculated. However, I've compromised...and potentially broken the lawy, but specifying the default value as a computed value e.g. `${publicDirectory}/hot`